articles/cdr-timescale-accounting/index.md

```diff
@@ -0,0 +1,166 @@
+---
+version: 1.0.0
+title: Comparing carbon removal approaches that act over different timescales
+authors:
+  - Claire Zarakas
+  - Grayson Badgley
+  - Freya Chay
+color: pink
+date: 09-14-2023
+summary: How should we compare carbon removal approaches that work on different parts of the carbon cycle at different timescales? We explore how understanding equilibration dynamics helps us compare different carbon removal approaches.
+quickLook: Explaining time and carbon cycle dynamics across carbon removal methods
+card: cdr-timescale-accounting
```

Comment:
It looks like the social card is using a different title -- is that intentional or should we update the text in the card? Or use the more concise title from the card?

![image](https://github.com/carbonplan/research/assets/12436887/383030c0-1bda-458a-9a9c-17df85bbcf62)

Reply:
Need to update the card!